### PR TITLE
[SERVICES-1053] Additional guideline for PR description and knowledge sources

### PR DESCRIPTION
**Pull request by ArturKlajnerok**
After reviewing our pull request guidelines I think they cover most important topics. I added a small proposition for usage of tasks list in descriptions when that can help with readability. 

Also linked two great knowledge sources for PR and Code Review. 

/cc @Wikia/services-team

### CODE REVIEW

File: Process/PullRequests.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ Each PR will:
           Example: “/cc @nmonterroso”.
        2. If the PR contains production SQL changes the @SQLReview tag should be
           used to notify the SQL review team.
+       3. If the PR cointain multiple tasks it can be useful to provide them in form of [Task List](https://github.com/blog/1825-task-lists-in-all-markdown-documents)
```
Comment (nmonterroso) [inline Process/PullRequests.md]: It might be worth mentioning that this could also potentially help with sharing work across multiple team members

Comment (ArturKlajnerok) [inline Process/PullRequests.md]: Let me know if this is addressing it properly : 92ee6a1

**Comment by drsnyder** [comment]
lgtm

**Comment by nmonterroso** [comment]
LGTM

**Pull request merged**